test/specs/options.spec.js

```diff
@@ -80,4 +82,71 @@ describe('options', function () {
     });
   });
 
+  describe('excludeOperators = ["equal"]', function () {
+    before(() => f = getFn('x === y',  {
+      excludeOperators: ['equal']
+    }));
+    it('does not warn for (null, number)', () => doesNotWarn(f, null, 1));
```

Comment:
Here (and below) it's enough to check that it does not warn for one combination of types. like you check in `mixed excludeOperators` section.